pyartnet/dmx_channel.py

```diff
@@ -104,18 +104,19 @@ def add_fade(self, target_values: Iterable[Union[int, FadeBase]],
             self.__fades[i] = fade
 
         if log.isEnabledFor(logging.DEBUG):
-            log._log(logging.DEBUG, f'Fade with {self.__step_max} steps:', [])
+            log.log(logging.DEBUG, f'Fade with {self.__step_max} steps:', [])
```

Comment:
_log was intentional because it's checking the log level in the line above

Reply:
Reverted